_includes/v23.1/prod-deployment/decommission-pre-flight-checks.md

```diff
@@ -0,0 +1,18 @@
+By default, CockroachDB will perform a set of "decommissioning pre-flight checks". That is, it will check if a node can successfully complete decommissioning given the current state of the cluster. If errors are detected that would result in the inability to complete node decommissioning, they will be printed to `STDERR` and the command will exit *without attempting to perform node decommissioning*. For example, ranges that are not yet fully upreplicated to nodes other than the one(s) being decommissioned will block the decommissioning process.
```

Comment:
> For example, ranges that are not yet fully upreplicated to nodes other than the one(s) being decommissioned will block the decommissioning process.

This is a bit misleading, as this example will only result in an error if run in the "strict" mode.  This may be somewhat confusing unfortunately.  A better one might be:
"For example, ranges that require a certain number of voting replicas in a region but do not have any available nodes in the region not already containing a replica will block the decommissioning process."

Reply:
Thanks, I've updated to use your example